src/components/SecondaryPanes/CommandBar.js

```diff
@@ -62,11 +62,13 @@ function formatKey(action) {
   return formatKeyShortcut(key);
 }
 
-function debugBtn(onClick, type, className, tooltip) {
+function debugBtn(onClick, type, className, tooltip, disabled) {
```

Comment:
hmm, small thing but maybe it's time to use an options object

```js
debugBtn({onClick, type, className, tooltip, disabled = false} = {})
```

This would, make the code more explicit, avoid needless `false` params...